code/modules/mob/mob.dm

```diff
@@ -70,98 +70,110 @@
 	t+= "<span class='notice'>N2O: [environment.sleeping_agent] \n</span>"
 	t+= "<span class='notice'>Agent B: [environment.agent_b] \n</span>"
 
-	usr.show_message(t, 1)
+	to_chat(usr, t)
 
-/mob/proc/show_message(msg, type, alt, alt_type)//Message, type of message (1 or 2), alternative message, alt message type (1 or 2)
 
-	if(!client)	return
+/// Show a message to this mob (MSG_VISIBLE or MSG_AUDIBLE)
+/mob/proc/show_message(msg, type, alt_msg, alt_type)//Message, type of message, alternative message, alt message type
+	if(!client)
+		return
 
 	if(type)
-		if(type & 1 && !has_vision(information_only=TRUE))//Vision related
-			if(!( alt ))
+		if((type & MSG_VISIBLE) && !has_vision(TRUE))//Vision related
+			if(!alt_msg)
 				return
-			else
-				msg = alt
-				type = alt_type
-		if(type & 2 && !can_hear())//Hearing related
-			if(!( alt ))
+			msg = alt_msg
+			type = alt_type
```

Comment:
```suggestion
			type = alt_type
			alt_msg = null // alternative is now the main solution
			alt_type = null
```
Doing it like this will tell later code that you don't have an alternative. `msg` is the final stand.
(the comment can be changed ofc)